app/assets/stylesheets/local/custom.scss

```diff
@@ -83,3 +83,70 @@ $app-blue-button-colour: govuk-colour("blue");
 .autocomplete__wrapper * {
   @include govuk-typography-common();
 }
+
+[aria-sort] .sorting,
```

Comment:
As this is quite a lot of custom CSS, perhaps we can create a separate `sorting.scss` file and `@import` it from the application.scss file, similar to what we do with some other big chunks of styling and custom components like the info_card, or the suggestions.